Testscripts/Windows/VERIFY-XDP-MTU.ps1

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+<#
+.Description
+    This script deploys the VM, verify XDP working with various MTU sizes (1500, 2000, 3506) which are easily configurable in XML.
+    Also, it will verify error caught by kernel "hv_netvsc" for MTU greater than Maximum MTU on Azure.
+#>
+
+param([object] $AllVmData,
+    [object] $CurrentTestData)
+
+$MIN_KERNEL_VERSION = "5.6"
+$iFaceName = "eth1"

```

Comment:
Can you read this interface name from system?

Reply:
This is a synthetic NIC name. This is going to be same in Azure environment.